src/main/jbake/templates/navigation.ftl

```diff
@@ -20,20 +20,41 @@
               <a href="<#if (content.rootpath)??>${content.rootpath}<#else></#if>docs/overview/getting-started.html">Getting
                   Started</a>
             </li>
+                <li class="dropdown">
```

Comment:
Minor - could be sorted later, but the indentation of this new block needs to be sorted.